### PR TITLE
BVPS-334 Seperate Home and Property Pages

### PR DESCRIPTION
**Pull request by emilymendelson**
https://deloittedigital.atlassian.net/browse/BVPS-334
- Adding in property-search route

### CODE REVIEW

File: components/Property/PropertyLayout/PropertyLayout.jsx [hunk 1]
```diff
@@ -6,18 +6,28 @@ import PropertyResultIcon from '../../../assets/svgs/PropertyResultIcon'
...
-        useState(true)
+    const [searchString, setSearchString] = useState()
+    const [showPropertySearchHeader, setShowPropertySearchHeader] = useState(true)
     const [propertySiteId, setPropertySiteId] = useState('')
     const [propertyAddress, setPropertyAddress] = useState('')
     const [searchResultLength, setSearchResultLength] = useState(0)
 
+    useEffect(() => {
+        let storedSearchString = sessionStorage.getItem("searchString")
+        console.log(storedSearchString)
```
Comment (hannah-macdonald1) [inline components/Property/PropertyLayout/PropertyLayout.jsx:23]: Console log here...

File: pages/home.js [hunk 2]
```diff
@@ -7,6 +7,10 @@ import { getUserInfo, getTokenInfo } from '../services/authentication/userAuthSe
 import PropertyLayout from '../components/Property/PropertyLayout'
 
 export default function Home(props) {
+    if (typeof window !== 'undefined') {
+        sessionStorage.setItem("searchString", '')
+        console.log('here')
```
Comment (hannah-macdonald1) [inline pages/home.js:12]: Ditto.

**Review by hannah-macdonald1 — CHANGES_REQUESTED** [review]
A couple of console logs to remove here.

File: components/Property/PropertyLayout/PropertyLayout.jsx (hunk 1 above)
Comment (emilymendelson) [reply components/Property/PropertyLayout/PropertyLayout.jsx]: Thanks! Fixed :)

**Review by hannah-macdonald1 — APPROVED** [review]

**Pull request merged**